src/models/StakeRequest.ts

```diff
@@ -0,0 +1,79 @@
+import { Address, Bytes32 } from '../types';
+
+export default class StakeRequest {
+  private _id: string;
```

Comment:
By not adding a `readonly` here, are we leaving the possibility to modify the states via some other functions ? 

Ex. from the event we get via TransactionHandlers, we create an instance. We then from our services modify the states of the same instance and pass it over to Repository classes to sync this state in Database ?

Reply:
variables are private so that can't be accessed outside the class.